bootstrap/kernel_bootstrap.php

```diff
@@ -25,9 +25,18 @@
 }
 
 $xml = new \SimpleXMLElement(file_get_contents($phpUnitFile));
-$kernelDir = $xml->php[0]->server[0]['value'];
 
-$kernelFile = $rootDir.'/'.$kernelDir.'/AppKernel.php';
+$envDir = $xml->xpath("//php/server[@name='KERNEL_DIR']");
+if (!count($envDir)) {
+    throw new \Exception(
+        'Kernel path must be set via <server name"KERNEL_DIR" value="..."/>'
+    );
+}
+$envClass = $xml->xpath("//php/env[@name='KERNEL_CLASS']");
+
+$kernelClass = count($envClass) ? (string) $envClass[0]['value'] : 'AppKernel';
+$kernelNs = explode('\\', $kernelClass);
+$kernelFile = $rootDir.'/'.$envDir[0]['value'].'/'.array_pop($kernelNs).'.php';
```

Comment:
the normal assumption in symfony 4 is that the kernel can be autoloaded, so we should try to autoload KERNEL_CLASS

Reply:
I agree. May be `KERNEL_DIR` should be required only if `KERNEL_CLASS` was not set?

Reply:
@dbu please, see #183 - i updated `bootstrap/kernel_bootstrap.php` there, `KERNEL_DIR` is used only for BC there (when kernel class name was `AppKernel`)